int32 motion_state
